bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -44,6 +45,8 @@ pub struct ExecStep {
     pub bus_mapping_instance: Vec<OperationRef>,
     /// Error generated by this step
     pub error: Option<ExecError>,
+    /// Non existent account addresses
+    pub non_existent_accounts: HashSet<Address>,
```

Comment:
For now this looks fine. But IMO we will be able to mix both existant and non-existant accounts within a single HashSet and structure.